packages/serverless/src/environment.js

```diff
@@ -9,16 +9,7 @@ const instanaEndpointUrlEnvVar = 'INSTANA_ENDPOINT_URL';
 const instanaAgentKeyEnvVar = 'INSTANA_AGENT_KEY';
 const instanaZoneEnvVar = 'INSTANA_ZONE';
 
-/**
- * TODO:
- *
- * This module is called BEFORE the serverless packages is required - see preactivate.js.
- * At this stage, we don't have any logger initialized yet.
- * For now, we just assign the Node.js inbuild `console` instance.
- *
- * We have to move the validation into the initialization of the serverless packages.
- */
-const logger = console;
```

Comment:
This does not work, because we see debug logs when requiring e.g. the aws lambda pkg.